### DISCUSSION TITLE
Is there a WeChat chat group?

### DISCUSSION THREAD

**Discussion by mengjie09** [MLT-OSS/open-assistant-api#79] [General]
Is there a WeChat chat group?

**Comment by Tuanzi1015** [comment]
![image](https://github.com/user-attachments/assets/57ce7ae7-0cf9-4a0b-be31-38f4849220af)
FYI

**Reply by mengjie09** [reply]
不好意思，才看到，消息失效了，麻烦在发一个，谢谢；
